homeassistant/components/channels/media_player.py

```diff
@@ -115,6 +144,30 @@ def service_handler(service):
         DOMAIN, SERVICE_SEEK_BY, service_handler, schema=CHANNELS_SEEK_BY_SCHEMA
     )
 
+    hass.services.register(
```

Comment:
You should register these as entity services. See Sonos for an example: https://github.com/home-assistant/core/blob/dev/homeassistant/components/sonos/media_player.py#L256-L264

Reply:
This component is a bit old school and doesn’t have that function. 

One portion of this pull, that wasn’t submitted, was setting it up with a config entry. While I got the UI going, the entities were never set up. I assumed it was for lack of implementing some of these (new to me) functions. 

Would implementing these get this component set up as entities?